application/apps/indexer/dlt/src/dlt_pcap.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2019 E.S.R.Labs. All rights reserved.
+//
+// NOTICE:  All information contained herein is, and remains
+// the property of E.S.R.Labs and its suppliers, if any.
+// The intellectual and technical concepts contained herein are
+// proprietary to E.S.R.Labs and its suppliers and may be covered
+// by German and Foreign Patents, patents in process, and are protected
+// by trade secret or copyright law.
+// Dissemination of this information or reproduction of this material
+// is strictly forbidden unless prior written permission is obtained
+// from E.S.R.Labs.
+
+use crate::dlt::{StorageHeader, DltTimeStamp};
+use pcap::Capture;
+use std::path::Path;
+use std::fs::File;
+use std::io::Write;
+use std::error;
+
+pub fn convert_pcap_to_dlt(pcap_path: &Path, dlt_path: &Path, port: u16) -> Result<(), Box<dyn error::Error>> {
+    let mut cap = Capture::from_file(pcap_path).unwrap();
```

Comment:
avoid `unwrap` since it might panic.
you already return a `Result` so propagate errors accordingly